lncfg/address.go

```diff
@@ -243,6 +243,10 @@ func ParseLNAddressString(strAddress string, defaultPort string,
 // a port, then we'll assume that the user is using the short cut to specify a
 // localhost:port address.
 func verifyPort(address string, defaultPort string) string {
+	if address == "" {
```

Comment:
Reminder to revert as this may get in the way of listening on both v4 and v6 for dual stack hosts.